src/main/kotlin/com/github/shiraji/ipgwizard/IPGWizardBuilder.kt

```diff
@@ -151,7 +151,8 @@ class IPGWizardBuilder : AbstractExternalModuleBuilder<GradleProjectSettings>(Pr
     }
 
     private fun setupSourceDirectory(modifiableRootModel: ModifiableRootModel, modelContentRootDir: VirtualFile) {
-        val resourceRootPath = "${modelContentRootDir.path}/src/main/java"
+        val languageDirectoryName = if (language == IPGWizardSupportLanguageStep.Language.KOTLIN.name) "kotlin" else "java"
+        val resourceRootPath = "${modelContentRootDir.path}/src/main/${languageDirectoryName}"
```

Comment:
I'm not sure this is good...but `IPGWizardBuilder#language` can be lower String.

https://github.com/shiraji/intellij-plugin-with-gradle-wizard/pull/18/files#diff-5b2793fd7ff4ea43146667290a4ff66bR50

```java
builder.setLanguage(Language.values()[languageComboBox.getSelectedIndex()].toString().toLowerCase());
```

and now

```kotlin
val resourceRootPath = "${modelContentRootDir.path}/src/main/${languageDirectoryName}"
```

will be

```kotlin
val resourceRootPath = "${modelContentRootDir.path}/src/main/$language"
```

And then fix template
https://github.com/shiraji/intellij-plugin-with-gradle-wizard/blob/1.1.0/src/main/resources/fileTemplates/internal/IPGWizard_Gradle%20Build%20Script.gradle.ft#L1

```
#if (${LANGUAGE} && ${LANGUAGE} == "kotlin")
```

https://github.com/shiraji/intellij-plugin-with-gradle-wizard/blob/1.1.0/src/main/resources/fileTemplates/internal/IPGWizard_Gradle%20Build%20Script.gradle.ft#L19-L22

```
apply plugin: '${LANGUAGE}'
```

https://github.com/shiraji/intellij-plugin-with-gradle-wizard/blob/1.1.0/src/main/resources/fileTemplates/internal/IPGWizard_Gradle%20Build%20Script.gradle.ft#L72

```
#if (${LANGUAGE} && ${LANGUAGE} == "kotlin")
```

What do you think?